src/main/java/Main.java

```diff
@@ -73,7 +73,7 @@ public void actionPerformed(ActionEvent e) {
     }
 
     private static void setupGame() throws FileNotFoundException {
-        File loadFile = new File("map.txt");
+        File loadFile = new File("src/main/resources/map.txt");
```

Comment:
Adding relative path makes the expected location easier to figure out. Good change.